cluster-up/cluster/kind-k8s-sriov-1.17.0/config_sriov.sh

```diff
@@ -9,6 +9,9 @@ PF_COUNT_PER_NODE=${PF_COUNT_PER_NODE:-1}
 
 [ $PF_COUNT_PER_NODE -le 0 ] && echo "FATAL: PF_COUNT_PER_NODE must be a positive integer" >&2 && exit 1
 
+SRIOV_NODE_LABEL=${SRIOV_NODE_LABEL:-"sriov=true"}
+NODE_PFS_ANNOTATION=${NODE_PFS_ANNOTATION:-"node_pfs"}
```

Comment:
Why do you need the key to be overwritten? I could not find the reason in the commit.

Reply:
I added those so those variable will have default value and for visibility, we already depend on few env vars that you need to investigate where they were came from.